openlibrary/templates/covers/add.html

```diff
@@ -50,10 +50,26 @@
     </div>
     <div class="input">
         <input type="file" name="file" id="coverFile" value="" accept=".jpg, .jpeg, .gif, .png" required/>
-        <button type="submit" class="cta-btn cta-btn--vanilla">$_("Upload")</button>
+        <button id='uploadButton1' type="submit" class="cta-btn cta-btn--vanilla">$_("Upload")</button>
     </div>
 </form>
 
+<form id="clipboardForm" class="ol-cover-form ol-cover-form--clipboard" method="post" enctype="multipart/form-data" action="$action">
+    <div class="label">
+        <label for="coverClipboard">$_("Or, paste an image from your clipboard")</label>
+    </div>
+    <div class="input">
+        <button type="button" onclick="pasteImage()" class="cta-btn cta-btn--vanilla" id="pasteButton">$_("No Image Pasted")</button>
```

Comment:
We try to avoid JS directly in our html like this ; attach this using a click handler in the JS

```suggestion
        <button type="button" class="cta-btn cta-btn--vanilla" id="pasteButton">$_("No Image Pasted")</button>
```